ui/src/app/settings/containers/Users/Users.scss

```diff
@@ -0,0 +1,29 @@
+@import "src/scss/vanilla";
+
+.user-list__button.p-button--base {
+  // Stop the icons from adding extra height to the row:
+  margin-top: -0.25rem;
+  margin-bottom: -0.25rem;
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+}
+
+.user-list .p-table-expanding__panel {
+  padding-top: 0;
+}
+
+.p-table-expanding th.user-list__cell--small,
+.p-table-expanding td.user-list__cell--small {
+  flex: 0 0 6rem;
+}
+
+.user-list__expanded-row {
+  background-color: #fff;
```

Comment:
This is also something we will want across all tables. Could you add to the table pattern locally here. For example: `p-table__row.is-active` then we could propose this to Vanilla.

Reply:
Could you use Vanilla colour variables please. `$color-x-light`

Reply:
Done.